community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStores.java

```diff
@@ -185,11 +186,15 @@ private void verifyRecordFormat()
         try
         {
             String expectedStoreVersion = recordFormats.storeVersion();
-            String actualStoreVersion = versionLongToString( getRecord( pageCache, neoStoreFileName, STORE_VERSION ) );
-            RecordFormats actualStoreFormat = RecordFormatSelector.selectForVersion( actualStoreVersion );
-            if ( !isCompatibleFormats( actualStoreFormat ) )
+            long record = getRecord( pageCache, neoStoreFileName, STORE_VERSION );
+            if ( record != MetaDataRecordFormat.FIELD_NOT_PRESENT )
```

Comment:
For the same reason that it is safe to skip it if the file does not exists.
In this case the file would have been created, but the database was shutdown before it was populated.